tests/functional/backend/ThemeControllerTest.php

```
@@ -169,15 +165,16 @@ public function testExtendTheme()
             sleep(1);
 
             // Check that we have the correct page header.
-            $header = $w->findElement(WebDriverBy::className('theme-editor-header'));
-            $this->assertEquals(
-                $header->getText(),
-                'Theme editor: vanilla_version_1',
-                $header->getText() . ' should equal "Theme editor: vanilla_version_1"'
+            // Wait for the page to redirect to the new theme editor.
+            $header = $w->wait(20)->until(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$header'. (undefined)

(UnusedLocalVariable)